qrisk/stats.py

```diff
@@ -499,57 +544,72 @@ def alpha_beta(returns, factor_returns):
     float
         Beta.
 
-"""
+    """
+    if len(returns) < 2:
+        return np.nan
 
-    ret_index = returns.index
-    beta, alpha = sp.sp.stats.linregress(factor_returns.loc[ret_index].values,
-                                      returns.values)[:2]
+    y = (returns - risk_free).loc[factor_returns.index].dropna()
+    x = (factor_returns - risk_free).loc[y.index].dropna()
+    y = y.loc[x.index]
+    beta, alpha = sp.stats.linregress(x.values, y.values)[:2]
 
-    return alpha * APPROX_BDAYS_PER_YEAR, beta
+    return alpha, beta
```

Comment:
It's not part of the typical alpha calculation. There is such a thing as annualized alpha but it's not the basic definition.